spec/models/user_spec.rb

```diff
@@ -87,4 +83,42 @@
       expect(user.token).to be_nil
     end
   end
+
+  describe "#connect_with_invitors" do
+    context "invited" do
+      let!(:user) { Fabricate(:user) }
```

Comment:
You don't need the bang here or the next one, since the `invite` instantiation will instantiate them.